tests/postgres/postgres.rs

```diff
@@ -2142,6 +2142,8 @@ create temporary table person(
     assert_eq!(people, p_query);
     Ok(())
 }
+
+#[allow(unused)]
```

Comment:
I'm not sure why this isn't marked as a test at all. Did you try running it?

If it's broken let's just mark it:
```suggestion
#[sqlx_macros::test]
#[ignore]
```

Reply:
The test passes; pr revision adds `#[sqlx_macros::test]` without the `ignore`.